cmsplugin_articles_ai/management/commands/publish_test_articles.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
I guess it's not that probable, yeah. For what it's worth, my i18nkit project prefixes commands with `ik_`...